src/runtime_init_memory.js

```diff
@@ -41,7 +41,10 @@ if (ENVIRONMENT_IS_PTHREAD) {
 #endif // ALLOW_MEMORY_GROWTH
 #if SHARED_MEMORY
       ,
-      'shared': true
+      'shared': true,
+#endif
+#if MEMORY64
+    'index': 'u64',
```

Comment:
Can you align this with `'shared'` above.    Also, move the trailing comma from line 43 up so that its on the `maximum` line.